core/src/test/scala/org/apache/spark/scheduler/BlacklistIntegrationSuite.scala

```diff
@@ -24,6 +24,7 @@ import org.apache.spark._
 class BlacklistIntegrationSuite extends SchedulerIntegrationSuite[MultiExecutorMockBackend]{
 
   val badHost = "host-0"
+  val duration = Duration(10, SECONDS)
```

Comment:
Pretty sure that such a long duration isn't really necessary, but I don't think it hurts to make it longer just in case.